test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceOptionsTests.cs

```diff
@@ -292,19 +290,15 @@ private async Task<Mock<RequestContext<SchemaCompareResult>>> SendAndValidateSch
                 {
                     TargetDatabaseName = targetDb.DatabaseName,
                     OperationId = schemaCompareOperation1.OperationId,
-                    ScriptFilePath = Path.Combine(folderPath, string.Concat(sourceDb.DatabaseName, "_", "Update.publish2.sql"))
                 };
 
                 SchemaCompareGenerateScriptOperation generateScriptOperation2 = new SchemaCompareGenerateScriptOperation(generateScriptParams2, schemaCompareOperation2.ComparisonResult);
-                generateScriptOperation2.Execute(TaskExecutionMode.Execute);
+                generateScriptOperation2.Execute();
 
-                // validate script
-                var filePath2 = Path.Combine(folderPath, string.Concat(sourceDb.DatabaseName, "_", "Update.publish2.sql"));
-                Assert.True(File.Exists(filePath2) && !string.IsNullOrEmpty(File.ReadAllText(filePath2)), "Should have differences differences");
+                // validate script generation succeeded
+                Assert.True(generateScriptOperation2.ScriptGenerationResult.Success);
```

Comment:
can we add little validation for content... may be that 1) content isn't empty 2) has a basic substring that we expect..?

Reply:
added back check if script is null or empty. I didn't add a check for a substring since the two tests that use this changed different objects.